src/services/itemMembership/test/fixtures/memberships.ts

```diff
@@ -20,14 +20,14 @@ export const saveMembership = ({
 
 export const saveItemAndMembership = async (options: {
   member: Member;
-  item?: Partial<Item>;
+  item?: Partial<DiscriminatedItem>;
```

Comment:
Here you have to use the `DiscriminatedItem` ? It is complaining when you try to use the `Item` entity ?